web-frontend/src/pages/Settings/SettingsSignRule.tsx

```diff
@@ -1,16 +1,25 @@
+import { useEffect, useMemo, useState } from 'react';
+
 import {
   Button,
   ButtonIcon,
   ButtonToggle,
   Clipboard,
   FetchingLine,
   toast,
+  Tooltip,
 } from '@massalabs/react-ui-kit';
-import { FiTrash2 } from 'react-icons/fi';
+import { maskAddress } from '@massalabs/react-ui-kit/src/lib/massa-react/utils';
```

Comment:
do not import path with "src"
`import { maskAddress } from '@massalabs/react-ui-kit'` should work. otherwise add needed export in ui-kit